app/src/main/java/com/kickstarter/services/apiresponses/ProjectStatsEnvelope.java

```diff
@@ -14,22 +14,17 @@
 @AutoParcel
 public abstract class ProjectStatsEnvelope implements Parcelable {
 
-  public enum ReferrerType {
```

Comment:
rather than removing this enum and using hard-coded strings, you can use a kotlin `enum class`! checkout [our koala context for some context](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/libs/KoalaContext.kt)